src/nextcloud/bin/nextcloud-cron

```diff
@@ -11,6 +11,17 @@ done
 printf "done\n"
 
 while true; do
-	run-php "$SNAP/htdocs/cron.php"
-	sleep 15m
+
+
+  # retrieve the current cronjob interval if the environement variable isn't set
+	if [ -z "$CRONJOB_INTERVAL" ]; then
+		CRONJOB_INTERVAL="$(cronjob_interval)"
+		export CRONJOB_INTERVAL
```

Comment:
It would be cleaner to always have the cronjob interval set and forgo the use of environment variables completely. Just set it to a default value on first run, like the other configurable items.